crates/modelardb_server/src/storage/uncompressed_data_manager.rs

```diff
@@ -204,10 +219,10 @@ impl UncompressedDataManager {
         Ok(())
     }
 
-    /// Insert `uncompressed_data_multivariate` into the in-memory buffer. If the data points are
-    /// from a multivariate time series, they are first split into multiple univariate time series.
-    /// These univariate time series are then inserted into the storage engine. Return [`String`] if
-    /// the channel or the metadata database could not be read from.
+    /// Insert `uncompressed_data_multivariate` into in-memory buffers. If the data points are from
+    /// a multivariate time series, they are first split into multiple univariate time series. These
+    /// univariate time series are then inserted into the storage engine. Return [`String`] if the
```

Comment:
detail: In (most?) other place, we use "Return*s* xxx if yyy". I don't have strong feelings for or against this :-)

Reply:
Fixed.